core/src/main/scala/cromwell/core/package.scala

```diff
@@ -10,4 +16,14 @@ package object core {
   type CallOutputs = Map[LocallyQualifiedName, JobOutput]
   type HostInputs = Map[String, WdlValue]
   type EvaluatedRuntimeAttributes = Map[String, WdlValue]
+
+  implicit def tryToErrorOr[A](trySomething: Try[A]): ErrorOr[A] = trySomething match {
+    case Success(options) => options.validNel
+    case Failure(err) => err.getMessage.invalidNel
+  }
```

Comment:
Two ToLs:
(1) I really don't like implicit conversions, could we have them as implicit enhancements instead? eg.
```
implicit class EnhancedErrorOr[A](eo: ErrorOr[A]) {
  def toTry: Try[A] = eo match { 
    ...
  }
}
implicit class EnhancedTry[A](t: Try[A]) {
  def toErrorOr: ErrorOr[A] = ...
}
```

(2) Can we put them somewhere that isn't implicitly brought into every class in the project (in case we don't want them)? E.g.
```
object ErrorOrExtensions {
  implicit class EnhancedErrorOr { ... }
  implicit class EnhancedTry { ... }
}
```
which can be imported only when needed, via: `import ErrorOrExtensions._`